home-geektime.html

```diff
@@ -0,0 +1,41 @@
+<div style="text-align: center; margin: 40px;">
+    <a href="http://techfest.geektime.co.il/">
+        <img src="img/geektime/devfest.png" style="max-width: 800px; width: 100%;" alt="Geektime Devfest 2016"/>
+    </a>
+</div>
+
+<h1>
+Welcome to the Geektime Devfest 2016 Riot!
+</h1>
+
+<p>
+This is where you can meet and chat with all the other attendees and speakers and ask them more questions following their sessions.
+</p>
+
+<p>
+To get started, please join some chat rooms!
+</p>
+
+<ul>
+    <li><a href="#/room/#geektime:matrix.org">The Devfest Lobby</a></li>
+    <li><a href="#/room/#geektime-philosophies:matrix.org">Devfest: Developer Philsophies track</a></li>
+    <li><a href="#/room/#geektime-architecture:matrix.org">Devfest: Architecture Showdown track</a></li>
+    <li><a href="#/room/#geektime-bigdata:matrix.org">Devfest: Big Data, Small Talks track</a></li>
+    <li><a href="#/room/#geektime-web:matrix.org">Devfest: The Web Age track</a></li>
+    <li><a href="#/room/#geektime-tinyscreens:matrix.org">Devfest: Tiny Screens track</a></li>
+    <li><a href="#/room/#geektime-beyondfingertips:matrix.org">Devfest: Beyond your Fingertips track</a></li>
+</ul>
+
+<p>
+To explore other rooms available on Matrix, click here: <a href="#/directory"><img style="vertical-align: middle" src="img/icons-directory.svg" width="25" height="25" alt="Room Directory"/></a>
+</p>
+
+<p>
+Riot is also available on <a href="https://itunes.apple.com/us/app/vector.im/id1083446067">iOS</a> and Android (<a href="https://play.google.com/store/apps/details?id=im.vector.alpha">Play Store</a> and <a href="https://f-droid.org/repository/browse/?fdid=im.vector.alpha">Fdroid</a> Store) as a 100% native mobile app.
+</p>
+
+<p>
+Riot is built on <a href="https://matrix.org">Matrix</a> - a new open protocol for interoperable and secure decentralised communication.
+<br/>
+To learn more, head over to <a href="https://matrix.org">Matrix.org</a> - and to better understand Riot, check out the official <a href="https://riot.im">Riot Website</a>.
+</p>
```

Comment:
It would be nice if this could be versioned somewhere else rather than living with the rest of the vector source (especially in the root dir). Ditto for the images.